packages/chatdown/package.json

```
@@ -35,7 +35,7 @@
     "@types/mocha": "^10.0.6",
     "@types/node": "^10.14.6",
     "@types/rimraf": "^2.0.2",
-    "chai": "^4.2.0",
+    "chai": "4.4.1",
```

As the ESM change started in version 5, we could use the `^` to accept future patched versions.
```suggestion
    "chai": "^4.4.1",
```

---

okay, fixed